### PR TITLE
Manipulate comment

### PR DESCRIPTION
**Pull request by yskmt**
Refactoring and rebase of https://github.com/michael-lazar/rtv/pull/59

Comment add/edit/delete are merged in one function `manipulate_comment`.

### CODE REVIEW

File: rtv/submission.py [hunk 1]
```diff
@@ -71,15 +71,14 @@ def open_link(self):
         url = self.content.get(-1)['permalink']
         open_browser(url)
 
-    @SubmissionController.register('c')
-    def add_comment(self):
+    def manipulate_comment(self, command):
```
Comment (michael-lazar) [inline rtv/submission.py]: I don't think that this currently works well as a single function. There are too many 
```python
if command == 'c':
    ...
elif command == 'e':
   ...
else
```
style blocks. You should factor out some of the "special cases" and place them back into their parent functions. I'll try to give some examples.

Comment (michael-lazar) [inline rtv/submission.py]: Actually the more I look at this, the more I think that it should just be 3 seperate functions.

**Comment by yskmt** [comment]
either way is fine for me. If I split them into three functions. Then it looks like my newest commit.

**Comment by michael-lazar** [comment]
Merged in #83

**Pull request closed without merging**